recipe/meta.yaml

```diff
@@ -50,6 +50,7 @@ requirements:
     - {{ compiler('fortran') }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - openssh  # [mpi == 'openmpi']
```

Comment:
Should this be added for `run` as well?

Reply:
I don't understand the source of the error.  Maybe it would be safer to do so.